pkg/helpers/config/userdata.yaml

```diff
@@ -1,17 +1,17 @@
 #cloud-config
-repo_update: true
```

Comment:
Updating at runtime just adds time which we are trying to minimize and is just "another thing" that could stop verifier from running and we already get the latest updates at build time. Let's focus on keeping a known good AMI and having it up-to-date?

Reply:
should be done in the reverse order imho : be in a position to have regular deliveries of AMI (and pulling updating one easily) before removing the update part

Reply:
We have just been refreshing the package listing, there's no associated `repo_upgrade: true`

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-cloud-init

Reply:
I'll raise this in a separate ticket - this change shouldn't be a part of this PR.